.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
your html and css appears to be having linter errors. Please address them

Reply:
Well done, but try to explain that projects with linter errors will be invalidated